tests/testdata/expected_addons/expected_output_global_config_configuration/Splunk_TA_UCCExample/bin/splunk_ta_uccexample_rh_oauth.py

```diff
@@ -12,6 +12,7 @@
 import splunk.admin as admin
 from solnlib import log
 from solnlib import conf_manager
+from solnlib.conf_manager import InvalidHostnameError,InvalidPortError
```

Comment:
```suggestion
from solnlib.conf_manager import InvalidHostnameError, InvalidPortError
```

Reply:
Made the changes, also updated `oauth.template`